src/amp.js

```diff
@@ -66,6 +66,8 @@ try {
     } finally {
       makeBodyVisible(document);
       perf.tick('e_is');
+      // NOTE(erwinm): move flush when have the new ticks in place.
```

Comment:
Please clarify the sentence. Not sure if this is "move *to* flush" or such? Also, we don't do "NOTE". It's either a TODO or not :)